Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/15_red.material

```diff
@@ -6,9 +6,9 @@
     "properties": {
         "baseColor": {
             "color": [
-                0.43414968252182007,
-                0.029556725174188614,
-                0.03955138474702835,
+                1.0,
+                0.0,
+                0.0038452735170722008,
```

Comment:
What's the source of this? It doesn't seem to be in line with red from the macbeth colorchecker - see https://en.wikipedia.org/wiki/ColorChecker

Reply:
+1 on that.
The old existing was wrong but so is this.

Reply:
I worked with Jonny to get the colors dialed correctly across the board. This involved setting the red that was mistakenly cast as green to its proper value, as well as going into each of the materials and modifying manually to accommodate for source of the color- when material was procedurally based, file texture assignments were removed entirely, and when colors were texture based, basecolor was set to white, texture map was assigned, and texture handling set to lerp (previously set to multiply)